sample-gominer.conf

```diff
@@ -1,6 +1,28 @@
 [Application Options]
 
 ; ------------------------------------------------------------------------------
+; General settings
+; ------------------------------------------------------------------------------
+;
+
+; Location of logfiles.
+; logdir=/some/path
+
+; Debug logging level.
+; Valid levels are {trace, debug, info, warn, error, critical}
+; You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set
+; log level for individual subsystems.  Use btcd --debuglevel=show to list
+; available subsystems.
+; debuglevel=info
+
+; Connect via a SOCKS5 proxy.  NOTE: Specifying a proxy will disable listening
+; for incoming connections unless listen addresses are provided via the 'listen'
+; option.
```

Comment:
Should probably drop the NOTE: portion since that is copy and paste that doesn't apply to gominer.